src/Analyzers/CSharp/Tests/UseExpressionBody/UseExpressionBodyForPropertiesAnalyzerTests.cs

```diff
@@ -428,7 +483,7 @@ public async Task TestOfferToConvertToBlockEvenIfExpressionBodyPreferredIfHasThr
 using System;
 class C
 {
-    int Goo [|=>|] throw new NotImplementedException();
+    {|IDE0025:int Goo => {|CS8059:throw new NotImplementedException()|};|}
```

Comment:
❔ What is CS8059?

Reply:
throw expressions isn't supported by C# 6.0. See https://github.com/dotnet/roslyn/issues/20362.

Reply:
This was an incorrectly-written test (it should have used `=> 0` instead of `=> throw ...`. When converting tests with errors that shouldn't have existed in the inputs or outputs, it's better to declare those via `ExpectedDiagnostics` so it's clear that they need to be corrected.

Reply:
No need to change this one this time, since you'll likely have a follow-up commit anyway to remove these. But it's a good approach for tracking fixes in future conversions.

Reply:
> This was an incorrectly-written test

@sharwell I believe the test is intentionally using throw based on your description in https://github.com/dotnet/roslyn/issues/20362. Or am I missing something?

Reply:
No you are correct, the markup syntax would be preferred for this case. 👍